### PR TITLE
Fix #11 - support from-title input

### PR DESCRIPTION
**Pull request by microdataxyz**
@michalvankodev Please review this pull request. It should solve #11 by providing an additional input `from-title`. If the value is true, it parses the `connectedIssues` from title and body. It also dedupe the connected issues to have better performance.

BTW, it's my first time to contribute GitHub Action repo. If you found anything improper, please tell me. Thanks.

Happy new year.🎆

### CODE REVIEW

**Review by michalvankodev — APPROVED** [review]
At first, I wanted you to do these changes, but I will probably have to do the publish myself so I can also compile and commit a new version.

> As the action is written in TypeScript it needs to be compiled and deployed as well. The deployment is done within the git repository itself as that's how github actions work. You need to run `npm run build` command to generate new build and include it in the commit.

I will probably have to do the release myself and include the new released version in the testing github action workflow.

File: src/index.ts [hunk 1]
```diff
@@ -14,9 +14,14 @@ function getInputAsArray(name: string, options?: core.InputOptions): string[] {
     .filter((x) => x !== '')
 }
 
+function getBooleanInput(name: string, options: core.InputOptions): boolean {
```
Comment (microdataxyz) [inline src/index.ts:17]: We use the old version of `@action/core`. It doesn't include the getBooleanInput API. I create one to have it.

**Comment by microdataxyz** [comment]
@michalvankodev I included the npm build result. BTW, I found the version of package.json is still 1.0.0. Should we update it to the latest version, like `1.1.1`, during the official release?

Honest speaking, it's my first time working on a GitHub action repo. I don't know how to test it locally.

File: src/index.ts (hunk 1 above)
Comment (michalvankodev) [reply src/index.ts:17]: We should just upgrade the `@action/core` dependency.

**Comment by microdataxyz** [comment]
@michalvankodev I don't have permission to run the workflow because of the first-time contribution. Maybe, it's your turn to finish this PR.
Screenshot 2023-01-10 090612

**Pull request merged**